examples/isomorphic/server/src/index.html

```diff
@@ -32,13 +42,17 @@
 
       updateScheduled = true
     }
-
     window.global_js = new GlobalJS()
 
-    function main () {
-      const { Client } = window.wasm_bindgen
+    import { Client, default as init } from './static/isomorphic_client.js';
```

Comment:
minor - can we remove the `./` here to make it a little more clear to those that are unfamiliar with ES modules that this is from the server and not from some local file?

Reply:
Was able to change `./` to `/`.  Looks like module specifiers require that relative references start with either `/`, `./`, or `../`.